images/teuto-cours/Dockerfile

```
@@ -0,0 +1,49 @@
+FROM archlinux:multilib-devel
+LABEL MAINTAINER="teuto.net"
+LABEL version="1.0"
+LABEL description="A small image with necessary tools for the k8s cours"
```

```suggestion
LABEL description="A small image with necessary tools for the k8s course"
```
The directory also needs to be renamed to `teuto-course`

---

ups. Renamed